**HYPOTHESIS TESTING**


Background:

Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.

Objective:

To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.
Data Provided:

•	The theoretical weekly operating cost model: W = $1,000 + $5X
•	Sample of 25 restaurants with a mean weekly cost of Rs. 3,050
•	Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units

Assignment Tasks:

1. State the Hypotheses statement:

2. Calculate the Test Statistic:

Use the following formula to calculate the test statistic (t):
where:

•	ˉxˉ = sample mean weekly cost (Rs. 3,050)
•	μ = theoretical mean weekly cost according to the cost model (W = $1,000 + $5X for X = 600 units)
•	σ = 5*25 units
•	n = sample size (25 restaurants)

3. Determine the Critical Value:

Using the alpha level of 5% (α = 0.05), determine the critical value from the standard normal (Z) distribution table.

4. Make a Decision:

Compare the test statistic with the critical value to decide whether to reject the null hypothesis.

5. Conclusion:

Based on the decision in step 4, conclude whether there is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.

Submission Guidelines:

•	Prepare python file detailing each step of your hypothesis testing process.
•	Include calculations for the test statistic and the critical value.
•	Provide a clear conclusion based on your analysis.



1. State the Hypotheses

Null Hypothesis (𝐻0): The mean weekly operating cost for the franchise is equal to the theoretical cost, i.e.,𝜇=1000+5⋅600.

Alternative Hypothesis (𝐻𝐴): The mean weekly operating cost for the franchise is higher than the theoretical cost, i.e.,𝜇>1000+5⋅600.

2. Calculate the Test Statistic

The theoretical weekly operating cost 𝜇0 is calculated as:

𝜇0=1000+5⋅600=4000

Next, we calculate the test statistic 𝑡 using the formula:  

 $$ 𝑡= { 𝑥ˉ − 𝜇0 \over  𝜎/√𝑛 } $$

where:

𝑥ˉ = sample mean weekly cost (Rs. 3050)

𝜇0 = theoretical mean weekly cost (Rs. 4000)

𝜎=5⋅25=125 (since the standard deviation of the number of units produced is 25 and each unit adds $5 to the cost)

𝑛 = sample size (25 restaurants)

Let's plug in the values:

$$t = \frac{3050 - 4000}{125 / \sqrt{25}} = \frac{-950}{125/5} = \frac{-950}{25} = -38$$

3. Determine the Critical Value

For a significance level (alpha) of 0.05 in a one-tailed test, the critical value from the standard normal (Z) distribution is:

In [1]:
from scipy.stats import norm

alpha = 0.05
critical_value = norm.ppf(1 - alpha)
print(f"Critical Value: {critical_value}")


Critical Value: 1.6448536269514722


4. Make a Decision
Compare the test statistic with the critical value:

If 𝑡  is less than the negative of the critical value, we reject the null hypothesis.

If 𝑡  is greater than or equal to the negative of the critical value, we fail to reject the null hypothesis.

**Since 𝑡 = −38 and the critical value is 1.645 , we reject the null hypothesis because −38<−1.645.**

5. Conclusion

Based on the analysis, we reject the null hypothesis and conclude that there is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the theoretical cost model suggests.

In [2]:
import numpy as np
from scipy.stats import norm

# Given data
sample_mean = 3050
theoretical_mean = 4000
std_dev = 5 * 25
n = 25

# Calculate the test statistic
t = (sample_mean - theoretical_mean) / (std_dev / np.sqrt(n))
print(f"Test Statistic: {t}")

# Determine the critical value
alpha = 0.05
critical_value = norm.ppf(1 - alpha)
print(f"Critical Value: {critical_value}")

# Conclusion
if t < -critical_value:
    conclusion = "Reject the null hypothesis - There is strong evidence to support the restaurant owners' claim."
else:
    conclusion = "Fail to reject the null hypothesis - There is no strong evidence to support the restaurant owners' claim."

print(conclusion)


Test Statistic: -38.0
Critical Value: 1.6448536269514722
Reject the null hypothesis - There is strong evidence to support the restaurant owners' claim.
